setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = strava_py
+version = 0.0.1
+description = Create artistic visualisations with your exercise data
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/marcusvolz/strava_py
+author = Marcus Volz
+license = MIT
+license_file = LICENSE
+classifiers =
```

Comment:
We can freely choose classifiers from https://pypi.org/classifiers/. These don't have a huge lot of use, but are listed on a package's PyPI page and can be browsed: https://pypi.org/project/Pillow/